src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -42,6 +42,8 @@ export default App.extend({
     'change:selectedActions': 'onChangeSelected',
   },
   onChangeStateSort() {
+    if (!this.isRunning()) return;
```

Comment:
Nice, this fixes it. Not seeing any console errors in my local sandbox.

And even if this function is returned without setting the comparator, the sort will still be applied when the the list view is done loading and rendered. Since the sort option was set in the state.